src/recovery/mod.rs

```diff
@@ -233,13 +254,71 @@ impl Recovery {
             self.hystart.start_round(pkt_num);
         }
 
+        // Pacing: Set the pacing rate if BBR is not used
+        if self.pacing_enabled &&
+            (self.cc_algo == CongestionControlAlgorithm::CUBIC ||
+                self.cc_algo == CongestionControlAlgorithm::Reno)
+        {
+            if let Some(srtt) = self.smoothed_rtt {
+                let rate = ((self.congestion_window as u128 * 1000000) /
```

Comment:
Why use `u128` and microseconds?

Reply:
(alternative would be to use https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_secs_f64 and truncate the result, but I see why `u128` is used here).

Reply:
As opposed to truncate micros to `u64` as it will probably be enough, or use https://doc.rust-lang.org/std/time/struct.Duration.html#method.as_secs_f64.

Reply:
I was being too cautious I believe, clearly u64 should be fine, thanks.